src/base/types/abstract/Maybe.jl

```diff
@@ -7,7 +7,7 @@ A nice name for "nullable" type.
 const Maybe{T} = Union{Nothing,T}
```

Comment:
Q: why no DSE here?

Reply:
I did not DSE anything constants, not sure they have something for it?

Reply:
ah this is the case of "constants docs", nvm.